mantle/kola/harness.go

```diff
@@ -272,6 +273,77 @@ func testRequiresInternet(test *register.Test) bool {
 	return false
 }
 
+type DenyListObj struct {
+	Pattern string   `yaml:"pattern"`
+	Tracker string   `yaml:"tracker"`
+	Streams []string `yaml:"streams"`
+	Arches  []string `yaml:"arches"`
+}
+
+type ManifestData struct {
+	AddCommitMetadata struct {
+		FcosStream string `yaml:"fedora-coreos.stream"`
+	} `yaml:"add-commit-metadata"`
+}
+
+func parseDenyListYaml() error {
+	var objs []DenyListObj
+
+	// Parse kola-denylist into structs
+	denyListFile, err := ioutil.ReadFile("src/config/kola-denylist.yaml")
+
+	if os.IsNotExist(err) {
```

Comment:
```suggestion
	denyListFile, err := ioutil.ReadFile("src/config/kola-denylist.yaml")
	if os.IsNotExist(err) {
```

I would delete the line between the statement and the error processing (grouping them together).